### PR TITLE
fix build and broken imports

### PR DESCRIPTION
**Pull request by pngwn**
## Description

Please include a concise summary, in clear English, of the changes in this pull request. If it closes an issue, please mention it here.

Closes: #(issue)

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: js/statustracker/static/index.svelte [hunk 1]
```diff
@@ -47,7 +47,7 @@
 <script lang="ts">
 	import { onDestroy } from "svelte";
 	// TODO: revisit this
-	import { app_state } from "../../app/src/stores";
+
```
Comment (freddyaboulton) [inline js/statustracker/static/index.svelte:50]: I guess we can finally delete the TODO comment 😂

Comment (pngwn) [reply js/statustracker/static/index.svelte:50]: I did wonder what that was actually referring to tbh, although i wrote it. Maybe it was indeed this.

File: js/statustracker/static/index.svelte [hunk 2]
```diff
@@ -47,7 +47,7 @@
 <script lang="ts">
 	import { onDestroy } from "svelte";
 	// TODO: revisit this
```
Comment (pngwn) [inline js/statustracker/static/index.svelte:49]:
```suggestion
```

**Comment by pngwn** [comment]
I think this is probably fine, works locally anyway so will merge and see.

**Pull request merged**